packages/core/src/ItemWithParams/ItemWithParams.ts

```diff
@@ -22,7 +22,7 @@ export const isItemWithParams = (item: ItemWithParams | unknown): item is ItemWi
 export class ItemWithParams {
   type = 'ItemWithParams' as const
   value: ItemValue;
-  params: Record<string, ParamValue>;
+  params: Record<string, unknown>;
```

Comment:
Consider keeping ParamValue here, as it somewhat documents what that generic param purpose is.
I see we have not yet updated ParamValue type to `unknown` though, so please also make changes there as needed

Reply:
You're right 💯, I didn't consider everything thoroughly. I'll revert the code changes related to `set params type`, and submit a new `PR` for that.